src/components/pages/daysoff.js

```diff
@@ -63,6 +64,27 @@ export default function Daysoff() {
             showToast(getText('daysoff.error.action'));
         }
     };
+
+    const numberOfSelectedFilter = () => {
+        let nbFilters = 0;
+        if (filterData.dayoffTypes.length !== 0) nbFilters += 1;
+        if (filterData.slackUsers.length !== 0) nbFilters += 1;
+        if (filterData.status.length !== 0) nbFilters += 1;
+        if (
+            DayJS(filterData.start).format('DD/MM/YYYY') !== DayJS().startOf('month').format('DD/MM/YYYY')
+            || DayJS(filterData.end).format('DD/MM/YYYY') !== DayJS().endOf('month').format('DD/MM/YYYY')
+        ) {
+            nbFilters += 1;
+        }
+        return nbFilters;
+    };
+
+    const isFilterActive = () => (numberOfSelectedFilter() !== 0);
```

Comment:
* This should be placed in the `filter` context
* It should be a state (boolean)
* It should be updated any time filter data is updated (useEffect)